test/unit/spec/common/jobManager-Spec.js

```diff
@@ -2238,20 +2468,22 @@ define([
 
         describe('cancelBatchJob', () => {
             beforeEach(function () {
-                this.jobManagerInstance = createJobManagerInstance(
-                    this,
-                    BatchMixin(JobManagerCore)
-                );
+                createBatchMixinJobManager(this);
             });
             it('cancels the current batch parent job', function () {
-                spyOn(this.bus, 'emit');
                 const allBatchJobIds = JobsData.allJobsWithBatchParent.map((jobState) => {
                     return jobState.job_id;
                 });
+
+                // ensure the request loop is activated
+                this.jobManagerInstance._initJobs({ batchId: BATCH_ID });
+                expect(this.jobManagerInstance.looper.requestLoop).toBeDefined();
+                expect(this.jobManagerInstance.listeners).toBeDefined();
```

Comment:
added in some extra tests to ensure that things get cleared up properly after cancelling or resetting a batch job